chain/chain/src/tests/gc.rs

```diff
@@ -661,7 +663,14 @@ fn test_fork_far_away_from_epoch_end() {
 
     // GC execution
     chain1
-        .clear_data(tries1.clone(), &GCConfig { gc_blocks_limit: 100, ..GCConfig::default() })
+        .clear_data(
+            tries1.clone(),
+            &GCConfig {
+                gc_blocks_limit: 100,
+                gc_fork_clean_step: fork_clean_step,
+                ..GCConfig::default()
```

Comment:
```suggestion
```

That way, if we add a new field to config, we'd need to think what value should the test use.